data/saas/config/shippo_config.yml

```diff
@@ -0,0 +1,52 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Shippo SaaS Config
```

Comment:
Can you rename this to `Shippo`? We won't have the saas_connector_registry very soon and this is where we will pull the name to use in the UI?